src/config/country.rs

```diff
@@ -0,0 +1,53 @@
+use std::{
+    fmt::{Display, Formatter},
+    str::FromStr,
+};
+
+/// All the possible countries in which the application can run.
+#[derive(PartialEq, Eq, Debug, Clone)]
+pub enum Country {
+    Unknown,
```

Comment:
I don't love the idea of services not setting the country information - we can make it mandatory here, and maybe should (you could perhaps even switch the builder to use a type-state pattern, like `SubscriberConfigBuilder<Formatter, State>`, and then have a state of `WithCountry` and only implement `build` on `SubscriberConfigBuilder<T, WithCountry>` to force users to set it before the type system lets them call `build`)

Reply:
I like it :+1: